src/main/shared/bullet/bullet-model.ts

```diff
@@ -1,6 +1,10 @@
-import * as Phaser from 'phaser';
+export interface SimpleVector {
```

Comment:
It could be simply Vector or Vector2

Reply:
Done
Was trying to distinguish Phaser's Vector2 from our one